specification/azsadmin/resource-manager/containerservice/Microsoft.ContainerService.Admin/stable/2019-11-01/managedClusters.json

```diff
@@ -0,0 +1,153 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2019-11-01",
+    "title": "ContainerServiceAdminClient",
+    "description": "Admin Container Service Management Client."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.ContainerService.Admin/locations/{location}/managedClusters": {
+      "get": {
+        "x-ms-examples": {
+          "Returns a list of managed clusters at the given location.": {
+            "$ref": "./examples/managedClusters/List.json"
+          }
+        },
+        "tags": [
+          "ContainerServices"
+        ],
+        "description": "Returns a list of managed clusters present in all tenant location.",
+        "operationId": "ContainerServices_List",
+        "parameters": [
+          {
+            "$ref": "../../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../../common-types/resource-management/v2/types.json#/parameters/LocationParameter"
+          },
+          {
+            "$ref": "../../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK -- The list of managed clusters has been returned.",
+            "schema": {
+              "$ref": "#/definitions/ContainerServicesList"
+            }
+          },
+          "default": {
+            "description": "Error Response.",
+            "schema": {
+              "$ref": "../../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ContainerServiceProperties": {
+      "description": "Container service properties.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "Container service id.",
+          "type": "string",
+          "readOnly": true
+        },
+        "name": {
+          "description": "Container service name.",
+          "type": "string",
+          "readOnly": true
+        },
+        "location": {
+          "description": "Container service location.",
+          "type": "string",
+          "readOnly": true
+        },
+        "subscriptionId": {
+          "description": "SubscriptionId where container service is present.",
+          "type": "string",
+          "readOnly": true
+        },
+        "creationDate": {
```

Comment:
if creation/lastmodifeddate info is necessary it must utilize the systemData contract:  https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/common-api-contracts.md#system-metadata-for-all-azure-resources

Reply:
Its a location based proxy only resource, not a tracked top level resource